Vagrantfile

```diff
@@ -59,6 +59,12 @@ fi
 
 ## install openvswitch and enable ovsdb-server to listen for incoming requests
 #(apt-get install -y openvswitch-switch > /dev/null) || exit 1
+(wget -nv -O ovs-common.deb https://cisco.box.com/shared/static/v1dvgoboo5zgqrtn6tu27vxeqtdo2bdl.deb &&
```

Comment:
We may want to pull and add this to the base image to avoid a pull every time. Also this will need http_proxy settings to be passed to `vagrant up` which will not work in automated jenkins build environment.